Formula/chunkwm.rb

```diff
@@ -0,0 +1,108 @@
+class Chunkwm < Formula
+  desc "Tiling window manager for macOS based on plugin architecture"
+  homepage "https://github.com/koekeishiya/chunkwm"
+  url "https://github.com/koekeishiya/chunkwm/archive/v0.2.28.tar.gz"
+  sha256 "917c6aa09521ff4b7463f731192eea6be6313f0fa4e585f63f85603d0993cf92"
+
+  head do
+    url "https://github.com/koekeishiya/chunkwm.git"
+  end
+
+  option "without-tiling", "Do not build tiling plugin."
+  option "without-ffm", "Do not build focus-follow-mouse plugin."
+  option "without-border", "Do not build border plugin."
+  option "with-transparency", "Build transparency plugin."
+
+  def install
+    # create plugins and example directories
+    (share/"plugins").mkpath
+    (share/"examples").mkpath
+
+    # install chunkwm
+    system "make", "install"
+    inreplace "#{buildpath}/examples/chunkwmrc", "~/.chunkwm_plugins", "$(brew --prefix chunkwm)/share/plugins"
+    bin.install "#{buildpath}/bin/chunkwm"
+    (share/"examples").install "#{buildpath}/examples/chunkwmrc"
+
+    # install chunkc
+    system "make", "--directory", "src/chunkc"
+    bin.install "#{buildpath}/src/chunkc/bin/chunkc"
+
+    # install tiling plugin
+    if build.with? "tiling"
+      system "make", "install", "--directory", "src/plugins/tiling"
+      (share/"plugins").install "#{buildpath}/plugins/tiling.so"
+      (share/"examples").install "#{buildpath}/src/plugins/tiling/examples/khdrc"
+    end
+
+    # install ffm plugin
+    if build.with? "ffm"
+      system "make", "install", "--directory", "src/plugins/ffm"
+      (share/"plugins").install "#{buildpath}/plugins/ffm.so"
+    end
+
+    # install border plugin
+    if build.with? "border"
+      system "make", "install", "--directory", "src/plugins/border"
+      (share/"plugins").install "#{buildpath}/plugins/border.so"
+    end
+
+    # install transparency plugin
+    if build.with? "transparency"
+      system "make", "install", "--directory", "src/plugins/transparency"
+      (share/"plugins").install "#{buildpath}/plugins/transparency.so"
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    Copy the example configuration into your home directory:
+      cp #{share}/examples/chunkwmrc ~/.chunkwmrc
+
+    Opening chunkwm will prompt for Accessibility API permissions. After access has been granted, the application must be restarted.
+      brew services restart chunkwm
+
+    This has to be done after every update to chunkwm, unless you codesign the binary with self-signed certificate before restarting
+      Create code signing certificate named "chunkwm-cert" using Keychain Access.app
+      codesign -fs "chunkwm-cert" #{bin}/chunkwm
+
+    For keybindings install and configure https://github.com/koekeishiya/khd.
+    chunkwm provides an example khd configuration file for easier setup:
+      cp #{share}/examples/khdrc ~/.khdrc
+    EOS
+  end
+
+  plist_options :startup => false
```

Comment:
I just meant that IIRC `plist_options :startup => false` is redundant because `false` is the default setting on `:startup`?